edk2toollib/log/junit_report_format.py

```diff
@@ -76,15 +77,17 @@ def SetSkipped(self):
         self.Time = time.time() - self._StartTime
 
     def LogStdOut(self, msg):
-        self.StdOut += msg.strip() + "\n "
+        self.StdOut += escape(msg.strip()) + "\n "
 
     def LogStdError(self, msg):
-        self.StdErr += msg.strip() + "\n "
+        self.StdErr += escape(msg.strip()) + "\n "
 
     def Output(self, outstream):
         outstream.write('<testcase classname="{0}" name="{1}" time="{2}">'.format(self.ClassName, self.Name, self.Time))
         if self.Status == JunitReportTestCase.SKIPPED:
-            outstream.write('<skipped />')
+            outstream.write('<skipped message="skipped test because of configuration" type="skipped">')
```

Comment:
I think i would suggest changing this.  A message like this isn't helpful as it is the same for every skipped case.  If you want to support adding a message then i think it needs to be fully plumbed thru so it can be caller controlled.

Reply:
Removed the message